DashAI/back/explainability/explainers/permutation_feature_importance.py

```diff
@@ -0,0 +1,99 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+from datasets import DatasetDict
+from sklearn.inspection import permutation_importance
+
+from DashAI.back.explainability.global_explainer import BaseGlobalExplainer
+from DashAI.back.models import BaseModel
+
+
+class PermutationFeatureImportance(BaseGlobalExplainer):
+    """Permutation Feature Importance is a explanation method to asses the importance
+    of each feature in a model by evaluating how much the model's performance
+    decreases when the values of a specific feature are randomly shuffled.
+    """
+
+    COMPATIBLE_COMPONENTS = ["TabularClassificationTask"]
+
+    def __init__(
+        self,
+        model: BaseModel,
+        scoring: Union[str, List[str], None] = None,
+        n_repeats: int = 5,
+        random_state: Union[int, None] = None,
+        max_samples: int = 1,
+    ):
+        """Initialize a new instance of PermutationFeatureImportance explainer.
+
+        Parameters
+        ----------
+            model: BaseModel
+                Model to be explained
+            scoring: Union[str, List[str], None]
+                Scorer to evaluate how the perfomance of the model
+                changes when a particular feature is shuffled
+            n_repeats: int
+                Numer of times to permute a feature
+            random_state: Union[int, None]
+                Seed for  the random number generator to control the
+                permutations of each feature
+            max_samples: int
+                The number of samples to draw from the dataset to calculate
+                feature importance at each repetition
```

Comment:
The items of the section should start at the same level of the title (same for all other docstrings) ;).

```suggestion
        model: BaseModel
            Model to be explained
        scoring: Union[str, List[str], None]
            Scorer to evaluate how the perfomance of the model
            changes when a particular feature is shuffled
        n_repeats: int
            Numer of times to permute a feature
        random_state: Union[int, None]
            Seed for  the random number generator to control the
            permutations of each feature
        max_samples: int
            The number of samples to draw from the dataset to calculate
            feature importance at each repetition
```